src/components/ChatList/elements/ChannelContent.style.ts

```diff
@@ -10,7 +10,7 @@ export const channelContentStyles = StyleSheet.create({
     lineHeight: 22,
     flex: 1,
     paddingBottom: 2,
-    color: COLORS.white2
+    color: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change doesn't affect the visibility and contrast of the text against its background. If `COLORS.white2` was a lighter shade, it might have been used for better readability.

```diff
-    color: COLORS.white2
+    color: COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->